### DISCUSSION TITLE
PPOCRLabel的结果是否可以合并或拷贝到异地继续标记

### DISCUSSION THREAD

**Discussion by happysunguy841031** [PaddlePaddle/PaddleOCR#13714] [Q&A]
原本电脑标记到一半，因为电脑要换人了，就移到硬盘再拉到新的电脑，虽然《Label.txt》《fileState.txt》《Cache.cach》都还在，但是都需要从头标记。请问如果数据标记量过大，是否可以多人一起标记再整合到一个文件里面？

**Comment by GreatV** [comment]
把文件里面对应的地址改成新电脑里面应该就可以了。
